src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -41,6 +41,7 @@ public static partial class AssertExtensions
     public static partial class PlatformDetection
     {
         public static bool IsReflectionEmitSupported;
+        public static bool BackendSupportsAlpn { get { throw null; } }
```

Comment:
The `PlatformDetection` class is basically the "Backend".  So, this property  can be named "SupportsAlpn".

Reply:
Will change.